src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -350,6 +358,16 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void OpenUrlInDefaultBrowser(Execution::Context& context)
+    {
+        if (context.Contains(Execution::Data::ShowUrls)) {
+            std::vector<std::string>& vectorOfUrls = context.Get<Execution::Data::ShowUrls>();
+            for (auto url : vectorOfUrls)

```

Comment:
>auto [](start = 17, length = 4)

const auto& (or auto const& if you want to get ahead of the curve of us old timers and our west const).